bigframes/ml/forecasting.py

```diff
@@ -199,11 +200,12 @@ def _fit(
         Returns:
             ARIMAPlus: Fitted estimator.
         """
-        if X.columns.size != 1:
+
+        if not isinstance(X, (pd.Series, bpd.Series)) and X.columns.size != 1:
             raise ValueError(
                 "Time series timestamp input X must only contain 1 column."
             )
-        if y.columns.size != 1:
```

Comment:
to generalize, convert to DataFrame first, then check columns. Then you don't need different logic for Series and DF.